src/MeshData.cpp

```diff
@@ -32,8 +33,19 @@ MeshData::~MeshData()
 
 }
 
+float parse_float(const pugi::xml_attribute& attribute)
+{
+    if(std::strchr(attribute.value(), ',') != nullptr)
+    {
+        std::setlocale(LC_NUMERIC, "nl_NL.UTF8");
```

Comment:
This should always be en-US (it should always force it to that locale). The test I was referring to was that we should check if the locale of the system influences what we write (eg; set the locale to nl_NL, but the 3mf file should then still use en_US)

Reply:
That already can't happen, since all locales (including the one for numbers) are set to "C" (which also uses a dot for the decimal sep. and is arguably even more correct) in the construtor of `ThreeMFParser`.

According to git blame, 'past you' agrees with me :-)

This was more to solve what happens if we get a file that does use comma's. I thought about adding error-handling, but that would imply fixing that for all cases (there's some TODO's in the code even), which was a bit out of scope for  the testing/tooling session.

I think I'll 'fix' it for now so that it -instead of trying to parse it- does a hard crash if there's comma's in there (same as would happen for invalid xml or the like right now), then make a sticky on the CCB to dicuss if we should add more error-handling to libSavitar any time soon.